src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,90 @@
+import { Loader } from './Loader';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { useEffect, useState } from 'react';
+import { PersonType } from '../types';
+import { Person } from './Person';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<PersonType[] | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+  const { personSlug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(peopleFromServer => {
+        const preparedPeople = peopleFromServer.map(person => ({ ...person }));
+
+        preparedPeople.forEach(person => {
+          Object.assign(person, {
+            mother:
+              preparedPeople.find(
+                mother => mother.name === person.motherName,
+              ) || null,
+            father:
+              preparedPeople.find(
+                father => father.name === person.fatherName,
+              ) || null,
+          });
+        });
+
+        setPeople(preparedPeople);
+      })
+      .catch(() => setErrorMessage('Unable to load people from server'))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {errorMessage && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {errorMessage}
+            </p>
+          )}
+
+          {people && !people.length && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
+
+          {!errorMessage && people && people.length > 0 && (
```

Comment:
```suggestion
          {!errorMessage && !!people.length && (
```

The same comment as above, create variable